# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: /Volumes/Sources/hudson_svn/hudson/main/core/src/main/java/hudson/model
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: Project.java
--- Project.java Base (BASE)
+++ Project.java Locally Modified (Based On LOCAL)
@@ -143,8 +143,8 @@
     protected void submit( StaplerRequest req, StaplerResponse rsp ) throws IOException, ServletException, FormException {
         super.submit(req,rsp);
 
-        if(!Hudson.adminCheck(req,rsp))
-            return;
\ No newline at end of file
+//        if(!Hudson.adminCheck(req,rsp))
+//            return;
\ No newline at end of file
 
         req.setCharacterEncoding("UTF-8");
 
